lib/trino-aws/src/main/java/io/trino/aws/AwsCredentialsProviderFactory.java

```diff
@@ -67,15 +77,39 @@
  */
 public class AwsCredentialsProviderFactory
 {
+    private static final AwsRegionProviderChain REGION_PROVIDER_CHAIN = new AwsRegionProviderChain(
+            new DefaultAwsRegionProviderChain(),
+            // fallback:
+            new AwsRegionProvider()
+            {
+                @Override
+                public String getRegion()
+                {
+                    return US_EAST_1.getName();
```

Comment:
Why is this hardcoded to US_EAST_1?

Reply:
I basically copied this from one of the places that this code replaces. It was hard-coded there as well :) Do you mean it should be configurable, or there should be no fallback?